lib/db.js

```diff
@@ -171,5 +185,6 @@ export const dbPlaces = [
     temperature: 30,
     review: "Awe-inspiring natural wonder, best experienced up close.",
     stars: 9,
+    isFavorite: false,
```

Comment:
You don't need to makes this entry in each object. If it is not there, Javascript interprets it as false and you function adds it to the object when toggled true